graphql-error-types/src/main/java/com/netflix/graphql/types/errors/TypedGraphQLError.java

```diff
@@ -70,30 +70,84 @@ public ErrorClassification getErrorType() {
         return extensions;
     }
 
+    @Deprecated
```

Comment:
Should we add a comment that the static builders are not _thread-safe_?

Reply:
Java 8 doesn't have an argument to the annotation, so added javadoc.